kubernetes/apps/charts/postgresql-backup/values.yaml

```diff
@@ -0,0 +1,9 @@
+cronjob:
+  schedule: "15 * * * *"
```

Comment:
Just for my own understanding: this will backup every hour and will keep the last 36 hourly, the last 7 daily, and the last 52 weekly?

Reply:
To tell you the truth I'm not entirely certain of how the retention settings intersect. @themightychris would know better (I c/p'd those values from him)

Reply:
@atvaccaro correct, this `restic forget` command will keep the last 36 snapshots, whatever their timing (hourly + any manual runs we might do) and then at least 1 per day for the last 7 days and at least 1 per week for the last 52 weeks